servlet.py

```diff
@@ -39,6 +39,7 @@
 from tornado import httpclient
 from tornado import gen
 from tornado import escape
+import configparser
```

Comment:
These imports are a complete mess. Would you mind breaking them into two blocks, one standard libraries and another external libraries and sorting each alphabetically?

Reply:
Sorting? Yeah, sure. Standard vs. External? Not sure how to tell.

Reply:
I think only tornado is external.

Reply:
eh, 3rd-party library is probably a better term

Reply:
oh, typing and tornado are now. the others can all just go in one sorted block